cheatsheets/Transport_Layer_Protection_Cheat_Sheet.md

```diff
@@ -1,411 +1,214 @@
-- Use of Ephemeral Diffie-Hellman key exchange will protect confidentiality of the transmitted plaintext data even if the corresponding RSA or DSS server private key got compromised. An attacker would have to perform active man-in-the-middle attack at the time of the key exchange to be able to extract the transmitted plaintext. All modern browsers support this key exchange with the notable exception of Internet Explorer prior to Windows Vista.
-- Some ciphers like `ECDHE-RSA-AES256-SHA` are explicitly inluded in the cipher string that it also works with elder versions of openssl.
+## Use HTTP Strict Transport Security
 
-Additional information can be obtained within the:
-- [TLS 1.2 RFC 5246](https://tools.ietf.org/html/rfc5246).
-- [SSL Labs: 'SSL/TLS Deployment Best Practices'](https://www.ssllabs.com/projects/best-practices/index.html),.
-- [BSI: 'TR-02102 Part 2 (German)'](https://www.bsi.bund.de/SharedDocs/Downloads/DE/BSI/Publikationen/TechnischeRichtlinien/TR02102/BSI-TR-02102-2.pdf).
-- [ENISA: 'Algorithms, Key Sizes and Parameters Report'](http://www.enisa.europa.eu/activities/identity-and-trust/library/deliverables/algorithms-key-sizes-and-parameters-report).
-- [RFC 7525: Recommendations for Secure Use of Transport Layer Security (TLS) and Datagram Transport Layer Security (DTLS)](https://tools.ietf.org/html/rfc7525).
-- [FIPS 140-2 IG](http://csrc.nist.gov/groups/STM/cmvp/documents/fips140-2/FIPS1402IG.pdf).
+HTTP Strict Transport Security (HSTS) instructs the user's browser to always request the site over HTTPS, and also prevents the user from bypassing certificate warnings. See the [HTTP Strict Transport Security cheatsheet](https://www.owasp.org/index.php/HTTP_Strict_Transport_Security) for further information on implementing HSTS.
 
-### Rule - Support TLS-PSK and TLS-SRP for Mutual Authentication
+## Use Public Key Pinning
 
-When using a shared secret or password offer TLS-PSK (Pre-Shared Key) or TLS-SRP (Secure Remote Password), which are known as Password Authenticated Key Exchange (PAKEs). TLS-PSK and TLS-SRP properly bind the channel, which refers to the cryptographic binding between the outer tunnel and the inner authentication protocol. IANA currently reserves [79 PSK cipher suites](http://www.iana.org/assignments/tls-parameters/tls-parameters.xml#tls-parameters-3) and [9 SRP cipher suites](http://www.iana.org/assignments/tls-parameters/tls-parameters.xml#tls-parameters-3).
+HTTP Public Key Pinning (HPKP) is used to associate a specific certificate with a domain name, in order to prevent an attacker performing a man in the middle attack with a different (but trusted) certificate. See the [Certificate and Public Key Pinning cheatsheet](https://www.owasp.org/index.php/Certificate_and_Public_Key_Pinning) for further information in implementing HPKP.
```

Comment:
The link to HPKP is not a cheat cheet 😃 

It is a header that in the process of being removed, see [here](https://www.chromestatus.com/feature/5903385005916160).

Reply:
@righettod I've rewritten part of this section to discuss the current status of HPKP - what do you think of the new text? It's still supported by Firefox and has some legitimate uses, so I'm not too keen on the idea of removing it completely just because Chrome dropped support, but I'd welcome your view.

Reply:
You're right, Sorru, I was to quick to remove it due to Chrome removal so we will let it.